edgelet/edgelet-docker/src/module.rs

```diff
@@ -28,6 +28,15 @@ pub struct DockerModule<C: Connect> {
     config: DockerConfig,
 }
 
+impl<C> std::fmt::Debug for DockerModule<C>
+where
+    C: Connect,
+{
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("DockerModule").finish()
```

Comment:
Won't this just display "DockerModule { }" - do we care about the fields?

Reply:
All these `Debug` impls are there to satisfy `Result::unwrap_err`, since that requires the `Ok` variant to impl `Debug` in case `unwrap_err` needs to panic.

I couldn't `#[derive(Debug)]` any of them because their fields aren't `Debug` either. Eventually I would have had to modify the API traits in the Docker crate to also be bound on `Debug`, at which point I gave up and created this empty impl.

It wasn't getting used before, and even now it doesn't get used outside of tests. Even if the tests panic because something returned `Ok` instead of `Err`, I don't think the internals of the struct would be particularly useful; you already know which test failed and can look it up from there.

Reply:
Sounds good, thanks for the explanation.